CMakeLists.txt

```diff
@@ -1,14 +1,14 @@
 cmake_minimum_required(VERSION 3.5)
 
-list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
+include(cmake/cable/bootstrap.cmake)
+include(CableToolchains)
+include(CableCompilerSettings)
 
-set(CMAKE_TOOLCHAIN_FILE ${CMAKE_CURRENT_SOURCE_DIR}/cmake/toolchains/cxx11-pic.cmake CACHE FILEPATH "CMake toolchain file")
+cable_configure_toolchain(DEFAULT cxx11-pic)
```

Comment:
Isn't this getting the toolchain from cable instead? Seems not since I've removed it from the `cmake/toolchains` directory and it is failing now.

Reply:
I  taught Cable to also check `${PROJECT_SOURCE_DIR}/cmake/toolchains` dir.